azkaban-common/src/main/java/azkaban/executor/ExecutionOptions.java

```diff
@@ -176,27 +180,39 @@ public static ExecutionOptions createFromObject(final Object obj) {
    * Validate the ExecutionOption's Flow-Parameters against the application level Properties
    * @throws ServletException if any of the parameter is invalid
    */
-  public void validateFlowParameters(Props azProps)
+  public void validatePreprocessFlowParameters(Props azProps)
```

Comment:
please consider moving this method to `HttpRequestUtils` class

Reply:
I think it's related to a `ExecutionOption` object and its usage can be larger than just HTTP related. E.g. for execution created other than using the Ajax endpoint